### ISSUE TITLE
[NETSDKE2E]With NET 9 alpha SDK installed, the message that you are using a preview version of .NET should be displayed when building project in CLI.

### ISSUE THREAD

**Issue by vdanche** [dotnet/msbuild#9602]
**Affect build:**
 .NET 9.0.100-alpha.1.24053.14(runtime-9.0.0-alpha.1.24053.1) from [dotnet/installer: .NET SDK Installer (github.com)](https://github.com/dotnet/installer)
Affected Product Language:

Steps to reproduce:

1. in CLI, dotnet new console and then dotnet build
2. Check the message displaying in CLI.

**Expected Result:**
the message that you are using a preview version of .NET should be displayed.
**Actual Result:**
the message that you are using a preview version of .NET has been removed.
![image](https://github.com/dotnet/sdk/assets/65637796/8c1c3d0c-a6c9-4a45-b2be-4e4a36ae6c7a)

But in VS(17.9 perview 3 + 9.0.100-alpha.1.24053.14 SDK). create a net9.0 console and build in VS, the message that you are using a preview version of .NET is still displayed.
![image](https://github.com/dotnet/sdk/assets/65637796/9d97e265-2f2f-44db-beb9-8b2842dcec22)

<details><summary>dotnet --info</summary>
```
.NET SDK:
 Version:           9.0.100-alpha.1.24053.14
 Commit:            48543d2076
 Workload version:  9.0.100-manifests.ae1d05f3

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.20348
 OS Platform: Windows
 RID:         win-x64
 Base Path:   C:\Program Files\dotnet\sdk\9.0.100-alpha.1.24053.14\

.NET workloads installed:
 [maui-windows]
   Installation Source: VS 17.9.34502.94
   Manifest Version:    8.0.3/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.maui\8.0.3\WorkloadManifest.json
   Install Type:        FileBased

 [maccatalyst]
   Installation Source: VS 17.9.34502.94
   Manifest Version:    17.0.8500/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.maccatalyst\17.0.8500\WorkloadManifest.json
   Install Type:        FileBased

 [ios]
   Installation Source: VS 17.9.34502.94
   Manifest Version:    17.0.8500/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.ios\17.0.8500\WorkloadManifest.json
   Install Type:        FileBased

 [android]
   Installation Source: VS 17.9.34502.94
   Manifest Version:    34.0.52/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.android\34.0.52\WorkloadManifest.json
   Install Type:        FileBased

 [aspire]
   Installation Source: VS 17.9.34502.94
   Manifest Version:    8.0.0-preview.1.23557.2/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.aspire\8.0.0-preview.1.23557.2\WorkloadManifest.json
   Install Type:        FileBased


Host:
  Version:      9.0.0-alpha.1.24053.1
  Architecture: x64
  Commit:       cc26196708

.NET SDKs installed:
  8.0.200-preview.23620.12 [C:\Program Files\dotnet\sdk]
  9.0.100-alpha.1.24053.14 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 7.0.14 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 8.0.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 9.0.0-alpha.1.23620.2 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 7.0.14 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 8.0.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 9.0.0-alpha.1.24053.1 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 7.0.14 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 8.0.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 9.0.0-alpha.1.24053.2 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

Other architectures found:
  x86   [C:\Program Files (x86)\dotnet]
    registered at [HKLM\SOFTWARE\dotnet\Setup\InstalledVersions\x86\InstallLocation]

Environment variables:
  Not set

global.json file:
  Not found

Learn more:
  https://aka.ms/dotnet/info
```
</details>

**Comment by baronfel** [comment]
This is an expected result of the change in defaults to use Terminal Logger from the CLI. No 'Message' items currently display in Terminal Logger, only Warnings and Errors.

**Comment by rainersigwald** [comment]
We have a couple of allowlisted messages (see #9407) and I think we should add this one, too.

**Comment by baronfel** [comment]
Fair point. There's a history of some users (internals and externals) being annoyed by the message when it appears multiple times, though: https://github.com/dotnet/sdk/issues/10352

So if we did do anything custom here it might also be nice to dedupe these specific messages.

**Comment by rainersigwald** [comment]
Yeah I'd be totally on board with that . . . except it's harder than the two-liner change I already have :-P

**Comment by vdanche** [comment]
This issue has been fixed on 9.0.100-alpha.1.24060.43(runtime-9.0.0-alpha.1.24060.1), thanks
![image](https://github.com/dotnet/msbuild/assets/65637796/e1ed360e-1136-4532-8070-a1c9a7942208)
